### DISCUSSION TITLE
Supporting the HTML5 `<abbr>` tag for under-dotted explainers & screenreaders

### DISCUSSION THREAD

**Discussion by michaelosthege** [community/community#9681] [Other Feature Feedback, Questions, & Ideas]
The [`<abbr>`](https://www.w3docs.com/learn-html/html-abbr-tag.html) tag would be useful to explain abbreviations in READMEs or comments.

As suggested by https://github.com/isaacs/github/issues/739 and the README here, I'm creating a discussion to propose adding the support.

For example, with the following markdown:

```markdown
The GitHub action workflows are defined in <abbr title="Yet Another Markup Language">YAML</abbr> files.
```

Renderers like the Markdown preview in VSCode, and most browsers when they encounter the tag show a small tooltip:

![image](https://user-images.githubusercontent.com/5894642/148303727-0af3019b-40af-4c98-8128-a5d289051747.png)

**Comment by timon** [comment]
I would like to second this issue. So far I can't explain acronyms (not necessary familiar to the reader) with `abbr` and have to resort to hacks around hyperlinks, like 

```html 
<a href='#' title='Yet Another Markup Language'>YAML</a>
```

**Comment by Leollaine** [comment]
<a href='#' title='Yet Another Markup Language'>YAML</a>

**Comment by gibson042** [comment]
I just ran into this as well, and had to work around it: https://github.com/tc39/ecma402/pull/746/commits/fba7828bdfc2caba8d9588d5958aadc1d940ecd0

Please stop stripping out `<abbr>`!

**Comment by cyyynthia** [comment]
I hate to be the kind of person to do "bumps" on threads, but this has been asked 3 years ago and still no updates about this quite important accessibility feature that seems to be a very easy fix...  😕
